enclave-manager/server/server.go

```diff
@@ -241,14 +237,155 @@ func (c *WebServer) GetServices(ctx context.Context, req *connect.Request[kurtos
 		return nil, err
 	}
 
+	serviceInfoMapFromApicObj := serviceInfoMapFromAPIC.Msg.GetServiceInfo()
+
+	// we aren't in a cloud context, so we exit early
+	if !c.enforceAuth {
+		resp := &connect.Response[kurtosis_core_rpc_api_bindings.GetServicesResponse]{
+			Msg: &kurtosis_core_rpc_api_bindings.GetServicesResponse{
+				ServiceInfo: serviceInfoMapFromApicObj,
+			},
+		}
+		return resp, nil
+	}
+
+	cloudClient, err := c.createKurtosisCloudBackendClient(
```

Comment:
I thought the lock/unlock would be part of the APIC so it can come from the EM UI and also CLI.

Reply:
Discussed here: https://kurtosistech.slack.com/archives/C060Z6HDYR3/p1712329736940959?thread_ts=1712322257.564799&cid=C060Z6HDYR3